tests/test_cli.py

```
@@ -3286,5 +3309,175 @@ def mock_exists_side_effect(path):
+        from mmrelay.cli import handle_k8s_command
+
+        result = handle_k8s_command(self.mock_args)
+
+        self.assertEqual(result, 1)
+        mock_print.assert_any_call("\n\nCancelled.")
+
+    @patch("builtins.input")
+    @patch("builtins.print")
+    @patch("mmrelay.k8s_utils.prompt_for_config")
+    def test_handle_k8s_command_import_error(
+        self, mock_prompt, _mock_print, _mock_input
```

Ruff (0.14.13)

Unused method argument: `mock_prompt`

(ARG002)